fw/beef.c

```diff
@@ -381,7 +388,12 @@ void update_lighting(int8_t tt1_report,
 }
 
 void update_button_lighting(uint16_t led_state,
-                            timer* combo_lights_timer) {
+                            timer* combo_lights_timer,
+			    config current_config) {
+  if (current_config.disable_button_led) {
```

Comment:
Should we turn off the turntable's lights as well? The PWAN has a button combo that turns off all lights, should we implement that as a separate combo?

Reply:
I think we can include that here. The only issue is that It will only work for people using digital lights since regular ring lights are just connected straight to +5V. Those people will just have disconnect the cable :)